core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -262,12 +262,17 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
             }
           }
 
-          saveAction.map(_ => RequestComplete(StatusCodes.NoContent))
+          saveAction
         }
       }
     }
   }
 
+  def batchUpdateEntities(workspaceName: WorkspaceName, entityUpdates: Seq[EntityUpdateDefinition], upsert: Boolean = false): Future[PerRequestMessage] = {
+    batchUpdateEntitiesInternal(workspaceName, entityUpdates, upsert).map(_ => RequestComplete(StatusCodes.NoContent))
+  }
+
+
```

Comment:
I factored this out into two methods, so that the PFB-import code can call `batchUpdateEntitiesInternal` and get something other than a `PerRequestMessage` back.